vision/detect/detect_test.go

```diff
@@ -57,30 +55,34 @@ func TestDetect(t *testing.T) {
 		if tt.local == nil {
 			continue
 		}
-
-		var buf bytes.Buffer
-		err := tt.local(&buf, "../testdata/"+tt.path)
-		if err != nil {
-			t.Fatalf("Local %s(%q): got %v, want nil err", tt.name, tt.path, err)
-		}
-		if got := buf.String(); !strings.Contains(strings.ToLower(got), strings.ToLower(tt.wantContain)) {
-			t.Errorf("Local %s(%q): got %q, want to contain %q", tt.name, tt.path, got, tt.wantContain)
-		}
+		t.Run(tt.name+"/local", func(t *testing.T) {
+			t.Parallel()
+			var buf bytes.Buffer
+			err := tt.local(&buf, "../testdata/"+tt.path)
+			if err != nil {
+				t.Fatalf("Local %s(%q): got %v, want nil err", tt.name, tt.path, err)
+			}
+			if got := buf.String(); !strings.Contains(strings.ToLower(got), strings.ToLower(tt.wantContain)) {
```

Comment:
nit: may be nicer to do:

```
if got, wantContains := strings.ToLower(buf.String), strings.ToLower(tt.wantContains); !strings.Contains(got, wantContains) {
    ...
}
```

(making the assertion just about asserting, and putting all the "munge up the data into the way I want it" into the declaration part)

Reply:
Done.